src/pymorize/files.py

```diff
@@ -130,6 +130,13 @@ def create_filepath(ds, rule):
     institution = rule.get("institution", "AWI")
     grid = "gn"  # grid_type
     time_range = _filename_time_range(ds, rule)
+    # check if output sub-directory is needed
+    enable_output_subdirs = rule._pymorize_cfg.get("enable_output_subdirs", False)
+    if enable_output_subdirs:
+        subdirs = rule.controlled_vocabularies.subdir_path(
+            rule.global_attributes_set_on_rule()
+        )
+        out_dir = f"{out_dir}/{subdirs}"
```

Comment:
I am not sure this makes sense to me. The subdirectories have nothing to do with the controlled vocabularies, this is mixing up what each class is supposed to be. Controlled Vocabularies are just lists of what is allowed for a specific key, that shouldn't be related to where the files are stored...?